app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :posts, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_comment_counter
+    num = Post.find_by(id: post_id).comments_counter.to_i
+    Post.find_by(id: post_id).update(comments_counter: num + 1)
+  end
+end
```

Comment:
- [ ] Whenever I tried to create  a new post comment , an error it is thrown as describe below, Kindly fix it

![Capture](https://github.com/francksefu/WebApp/assets/80458199/b49bfcb5-935c-4b6f-922d-ea46898ca7fd)

Reply:
In my PR , I said , for create Comment and Like, write post with s like this:
Comment.create(posts: first_post, author: second_user, text: 'Hi')

Reply:
![image](https://github.com/francksefu/WebApp/assets/90141102/36a18153-2cf4-4f7e-99a2-9468508564dc)